src/lib/components/Toast.svelte

```diff
@@ -38,6 +38,7 @@
   let overflow: "scroll" | "truncate" | "clamp" | undefined;
   let position: ToastPosition | undefined;
   let icon: typeof SvelteComponent | undefined;
+  // TODO-colors: Can we remove this?
```

Comment:
probably no because of the toast top right that sometimes has to be forced in a particular theme

Reply:
thanks for the tip!